partials/cli-plugins/_insight-ingress-configuration.hbs.md

```diff
@@ -0,0 +1,32 @@
+<!-- Configure certificate and end point for insight cli when ingress is enabled -->
+
+To get a certificate, run:
+
+```bash
+kubectl get secret ingress-cert -n metadata-store -o json | jq -r '.data."ca.crt"' | base64 -d > insight-ca.crt
+```
+
+The endpoint host is set to `metadata-store.<ingress-domain>`,
+such as `metadata-store.example.domain.com`).
+
+This value matches the value of `ingress_domain`.
```

Comment:
It's a little unclear to me what `This value` means in this context. Can this be rephrased to make that less ambiguous?

Reply:
reworded, is it better?